### ISSUE TITLE
[BUG] iOS app shows Live Photo as well as seperate video file

### ISSUE THREAD

**Issue by ronbruins** [immich-app/immich#2665]
### The bug

Timeline view of the iOS app shows the live(motion) photo but also the video file separate. Uber search ‘motion photos’ it shows fine and also at the web it shows correct (even with the motion photo icon) 

### The OS that Immich Server is running on

Ubuntu 22.04

### Version of Immich Server

V1.60

### Version of Immich Mobile App

V1.60

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
Default
```

### Your .env content

```Shell
Default
```

### Reproduction steps

```bash
1.open timeline view in iOS app
2.
3.
...
```

**Comment by jrasm91** [comment]
Is this an issue?

**Comment by ronbruins** [comment]
Hi,Yes it is. And I found some more reproductive steps. When I upload via CLI and create album with —album option the following happens:  open iOS app and Live Photos show OK. When I click in ‘library’ where the generated album exists and go back to main screen I see the Live Photo and the video file. When I delete the album all looks good again.So it looks like the automatic creation of an album includes both the image and video file for live and doesn’t combine them there. This somehow is back inherited on the main page of the iOS app.So it seems to do with auto generated albums. However with the new library feature, I don’t know what has changed for this. Because now albums can’t be generated on scan, so the problem will not be seen. But issue 4209 is not fixed yet, so nothing to test right now. Regards,RonOn 27 Sep 2023, at 22:56, Jason Rasmussen ***@***.***> wrote:﻿
Is this an issue?

**Comment by jrasm91** [comment]
Hmm, can you help clarify a few things for me? Do you see a live photo properly linked on the web, but the same photo shows up as two individual files in the mobile app?

There is an existing bug in the current, released version of the immich server where live photos are not getting linked automatically. This has been fixed, but not released yet. I want to make sure you are not just experiencing this issue, but that when a live photo is linked properly on the server, there is still an issue with how the mobile app is displaying it, which I think is what the original issue was.

**Comment by ronbruins** [comment]
Hi, Web is fine. Only happens on mobile. Yeah I had the same issue with 1.79 where everything was a mess :) I also saw that it’s in the latest commits. It is definitely not that current issue. What I had was: - cli upload with —album option- open mobile app and all looked fine. - selected albums and opened an album and then the files were split - back to home page and files showed split as well. As long as I did it go to the album page the main page was fine. When I went to the album page it got messed up, but when deleting the album all was fine again. I can test with the latest release when 4209 is fixed to see if I get the same behavior still. However I believe I can’t automatically create albums when using the library feature, is it? I will try the upload then again, to reproduce the issue. And let you know. On 28 Sep 2023, at 06:00, Jason Rasmussen ***@***.***> wrote:﻿
Hmm, can you help clarify a few things for me? Do you see a live photo properly linked on the web, but the same photo shows up as two individual files in the mobile app?
There is an existing bug in the current, released version of the immich server where live photos are not getting linked automatically. This has been fixed, but not released yet. I want to make sure you are not just experiencing this issue, but that when a live photo is linked properly on the server, there is still an issue with how the mobile app is displaying it, which I think is what the original issue was.

**Comment by ronbruins** [comment]
This seems to be resolved too. I tested it and the behaviour is gone.

**Comment by ronbruins** [comment]
Issues has been resolved

**Comment by EK1288** [comment]
Hi, reopening this as I have experienced the same bug. 

As with @ronbruins I have used CLI with the album flag to get my photos into Immich. On Web all of my assets look correct but on iOS, my entire timeline has split images and live photos. 

I tested deleting and recreating one album with live images. Signed out of the iOS app, signed back in to force a timeline rebuild, and those images now render correctly (single asset) but all others still in CLI created albums do not. I then recreated the album by manually adding from my timeline and everything is rendering correctly. 

Immich Server: v1.109.2
Immich Mobile App: 1.109.0 build.164

**Comment by nickygerritsen** [comment]
I have the same issue, only on mobile everything is doubled with Live Photos, where I see both the image (with live data) and video.

What is the easiest way to fix this? I don't want to manually recreate all my albums.

If this means writing some SQL query, that's fine with me.

**Comment by alextran1502** [comment]
@nickygerritsen how did you upload those LivePhotos, are they from external library?

**Comment by nickygerritsen** [comment]
> @nickygerritsen how did you upload those LivePhotos, are they from external library?

I exported them from iPhoto with `osxphotos` and then used the CLI to mass import them

**Comment by thorsten-hehn** [comment]
Hi, I can observe exactly the same behaviour as described by @ronbruins and @EK1288 before. I'm using v1.123.0 both on server and mobile iOS app.

I have automatically created ~900 albums from external libraries using the Javacript from davidacampos ([Link](https://gist.github.com/davidacampos/47138edcdc3ac040c05e8f3d6f6a9224)). Three of them showed a broken album thumbnail. These albums contained several live pictures, whereas each of them were split into one static picture and a video with broken thumbnail. Interestingly, the split live pictures could only be observed in the mobile app, whereas it looked fine on the web  The broken album thumbnail could be observed both in the mobile app and the web.

After deleting the corresponding albums and manually recreating them and adding the pictures again, everything works properly (no broken thumbnails, no split live pictures). So, the automatic album creation process definitely is causing the split live pictures issue.

**Comment by EricChen1248** [comment]
I've been able to replicate what thorsten is experience. I have a custom script that also does something very similar to the external library album sync script. 

Previously, if I didn't handle live photo videos, and added both the image portion and the video portion to an album
* On the web, it's smart enough to still hide both images.
* On iOS mobile, it will show the video, but because it doesn't have a thumbnail, it will show the blank exclamation mark icon. Clicking into the image will load the video though.

Once I updated my script to preemptively handle live photos to only add the image part to an album now everything displays as expected. On both web and mobile, only one single image shows in the gallery with all thumbnails showing, and clicking into the image, it is able to play the video as a live photo.
